static/app/views/issueDetails/traceTimelineOrRelatedIssue.tsx

```diff
@@ -0,0 +1,52 @@
+import {Fragment} from 'react';
+import styled from '@emotion/styled';
+
+import {space} from 'sentry/styles/space';
+import type {Event} from 'sentry/types/event';
+import useRouteAnalyticsParams from 'sentry/utils/routeAnalytics/useRouteAnalyticsParams';
+
+import {TraceIssueEvent} from './traceTimeline/traceIssue';
+import {TraceLink} from './traceTimeline/traceLink';
+import {TraceTimeline} from './traceTimeline/traceTimeline';
+import {useTraceTimelineEvents} from './traceTimeline/useTraceTimelineEvents';
+
+export function TraceTimeLineOrRelatedIssue({event}: {event: Event}) {
+  // This is also called within the TraceTimeline component but caching will save a second call
+  const {isLoading, oneOtherIssueEvent} = useTraceTimelineEvents({
+    event,
+  });
+  let params = {};
```

Comment:
right now our settings allow you to set any property but in the future the type will be `{}` which will prevent this
```suggestion
  let params: Record<string, boolean> = {};
```